### DISCUSSION TITLE
Running YOLOV8 on non CUDA GPU

### DISCUSSION THREAD

**Discussion by samareshdatta** [ultralytics/ultralytics#10066] [Q&A]
CUDA GPU are expensive. How to run YOLOV8 training on non CUDA GPU such as RADEON GPUs ?

**Comment by pderrenger** [comment]
@samareshdatta hello! Thanks for reaching out. 😊 YOLOv8, like many deep learning models, is optimized for CUDA-enabled GPUs due to the extensive support and optimization available through NVIDIA's libraries. However, for non-CUDA GPUs such as Radeon, you might explore running YOLOv8 under ROCm, AMD's open-source platform for GPU computing. Please note that direct support for ROCm in YOLOv8 and PyTorch (which YOLOv8 uses) might be limited, and performance can vary compared to running on NVIDIA's CUDA GPUs.

Another alternative is to use cloud-based solutions or rental services that offer access to NVIDIA GPUs, which can be cost-effective without needing to invest in hardware upfront.

If you're specifically looking into training and inferencing with AMD GPUs, keeping an eye on updates from the ROCm platform and community contributions to YOLO might offer some pathways. Let's hope for broader support in the future! 🌟
